.github/workflows/publish-x86_64-pc-windows-wasm.yml

```
@@ -69,6 +67,7 @@ jobs:
       - name: Test built artifact
         shell: powershell
         run: |
+          cp ./noir/target/${{ matrix.target }}/release/nargo ~/.cargo/bin/

```

```suggestion
          cp ./noir/target/${{ matrix.target }}/release/nargo.exe ~/.cargo/bin/

```
I think windows adds `.exe`